tests/test_core.py

```diff
@@ -6958,27 +6959,26 @@ def test_webidl(self, mode, allow_memory_growth):
 
     # Export things on "TheModule". This matches the typical use pattern of the bound library
     # being used as Box2D.* or Ammo.*, and we cannot rely on "Module" being always present (closure may remove it).
-    self.emcc_args += ['-s', 'EXPORTED_FUNCTIONS=_malloc,_free', '--post-js', 'glue.js']
+    self.emcc_args += ['-s', 'EXPORTED_FUNCTIONS=_malloc,_free', '--post-js=glue.js', '--extern-post-js=post.js']
```

Comment:
maybe call it `extern-post.js` to avoid confusion?

Reply:
(as there is also an actual `--post-js` in addition to the `--extern-post-js`)

Reply:
Good idea!